charts/em-npa/values.template.yaml

```diff
@@ -0,0 +1,60 @@
+postgresql:
+  initdbScripts:
+    init.sql: |-
+      CREATE USER emnpa WITH PASSWORD 'emnpa';
+      CREATE DATABASE emnpa WITH OWNER = emnpa ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
```

Comment:
if you only have one DB you can just use the regular config:
https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.template.yaml#L18